kahuna/public/js/upload/jobs/required-metadata-editor.js

```diff
@@ -30,7 +31,7 @@ jobs.controller('RequiredMetadataEditorCtrl',
         var cleanMetadata = {};
         Object.keys(ctrl.metadata).forEach(key => {
             if (ctrl.metadata[key] !== ctrl.saveWhenChangedFrom[key]) {
-                cleanMetadata[key] = ctrl.metadata[key];
+                cleanMetadata[key] = ctrl.metadata[key] || '';
```

Comment:
I don't understand why this is here? If it's been edited, angular sets it to `''` no?